protos/feast/types/FeatureRow.proto

```diff
@@ -25,12 +25,6 @@ option java_package = "feast.types";
 option java_outer_classname = "FeatureRowProto";
 option go_package = "github.com/gojek/feast/protos/generated/go/feast/types";
 
-message FeatureRowKey {
```

Comment:
Will removing the `FeatureRowKey` affect how CoalesceRows functions?

Reply:
No, it's not used by anything except for the kafka source we never really needed it.

Coalesce always used it's own CoalesceKey proto defined in `ingestion/protoe/feast_ingestion/types/CoalesceKey.proto`.  It is internal to the ingestion project.

Reply:
Ok. Just clarifying. It looks good to me.